lib/ansible/modules/cloud/ovirt/ovirt_host.py

```diff
@@ -327,6 +327,9 @@ def update_check(self, entity):
             )
         )
 
+    def pre_create(self, entity):
+        self.start_event = self._connection.system_service().events_service().list(max=1)[0]
```

Comment:
Initilialize this early in main, becaose pre_create is not called in case of reinstall.